controllers/users.js

```diff
@@ -1,39 +1,57 @@
-const { users } = require('../database/users.js');
+import { ObjectId } from 'mongodb';
+import User from '../models/users.js';
 
-function addUser(req, res) {
-  users.push(req.body);
-  res.status(200).json({ message: 'User created' });
-}
+const mongoObjectId = function () {
```

Comment:
if you need to generate id so please use uuid. https://www.npmjs.com/package/uuid

Reply:
done